mlrun/render.py

```diff
@@ -96,20 +99,33 @@ def link_html(text, link=""):
 
 
 def artifacts_html(x, pathcol="path"):
-    if not x:
+    artifact_dicts, output_path = x
```

Comment:
we gotta have more description in this area, most of it wasn't been touched from 2019, please try to add more information